book/website/communication/dif-articles/data.md

```diff
@@ -0,0 +1,112 @@
+(cm-dif-articles-data)=
+# Data Papers
+
+(cm-dif-articles-data-what)=
+## What are Data Papers?
+
+The purpose of a data paper is to describe a dataset and make it more FAIR (findable, accessible, interoperable and reusable). 
+Therefore, it is different from a research article, it is not meant to report the findings of your data. 
+
+It is a peer-reviewed article focused on describing datasets and the circumstances of their collection - a searchable metadata document. 
+
+There are specific data journals for data papers such as Scientific Data or Biodiversity Data Journal. 
+Data journals are usually open access journals so a data paper does normally come with an article processing cost (APC), but this can vary considerably depending on the journal you choose. 
```

Comment:
```suggestion
Data journals are usually open access journals so a data paper does normally come with an article processing charge (APC), but this can vary considerably depending on the journal you choose. 
```